console-subscriber/src/init.rs

```diff
@@ -0,0 +1,81 @@
+use crate::TasksLayer;
+use std::thread;
+use tokio::runtime;
+use tracing_core::{subscriber::Subscriber, Dispatch};
+use tracing_subscriber::{fmt, layer::Layered, prelude::*, EnvFilter, Layer, Registry};
+
+type ConsoleSubscriberLayer =
+    Layered<TasksLayer, Layered<EnvFilter, Layered<fmt::Layer<Registry>, Registry>>>;
+
+/// Starts the console subscriber server on its own thread
```

Comment:
typo: missing period :)
```suggestion
/// Starts the console subscriber server on its own thread,
```

Reply:
addresssed in 572e4547927f77061b6727e89a385f5a2691b341